### DISCUSSION TITLE
An 80T encoder gear for 40T spindle gear

### DISCUSSION THREAD

**Discussion by mockendon** [kachurovskiy/nanoels#36] [Show and tell]
Another remix - my spindle has a tiny 40T gear so this is needed to fit in under the pulleys.  Reduced pulses per revolution but cheaper and quicker than trying to source a new spindle gear.  Just #define ENCODER_STEPS to half of the value for the encoder used (2:1 reduction).    The axle bore is a pilot size as I found it best to drill to size on the lathe to improve concentricity. 

https://www.thingiverse.com/thing:5156106

**Comment by kachurovskiy** [comment]
Matt, thank you, the thingiverse link seems broken though.

**Reply by mockendon** [reply]
No but github has broken the hyperlink.  Works if you paste the url

edit, fixed the hyperlink 😜
